mgmtsystem_nonconformity/__manifest__.py

```diff
@@ -20,6 +20,7 @@
         'views/mgmtsystem_severity.xml',
         'views/mgmtsystem_action.xml',
         'views/mgmtsystem_nonconformity_stage.xml',
+        'views/report_mgmtsystem_nonconformity.xml',
```

Comment:
```suggestion
        'reports/report_mgmtsystem_nonconformity.xml',
```

Reply:
@ivantodorovich I chose to put the file in the views folder because of the OCA Guidelines. Or did I get it wrong?

![image](https://user-images.githubusercontent.com/3595132/87057924-d7c27500-c1dd-11ea-8dc4-31411e0e4156.png)

Reply:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#17complete-structure

Reply:
Hmmm yeah, seems your right

Reply:
I'm slowly learning heheheh